app/models/miq_worker.rb

```diff
@@ -135,11 +144,11 @@ def self.sync_workers
       _log.info("Workers are being synchronized: Current #: [#{current}], Desired #: [#{desired}]")
 
       if desired > current && enough_resource_to_start_worker?
-        (desired - current).times { result[:adds] << start_worker.pid }
+        (current..desired-1).each { |instance| result[:adds] << start_worker(:instance_name => instance).pid }
```

Comment:
`(current...desired)` ?

Reply:
Alternately, `current.upto(desired)` might read a lot better

Reply:
`.upto(desired)` would lead to one too many workers, but yeah `...` is better than what I have